tests/sentry/api/endpoints/test_user_details.py

```diff
@@ -313,8 +314,8 @@ class UserDetailsStaffUpdateTest(UserDetailsTest):
     method = "put"
 
     @fixture(autouse=True)
-    def use_staff_feature_flag(self):
-        with Feature("auth:enterprise-staff-cookie"):
+    def use_staff_option(self):
+        with override_options({"staff.user-email-allowlist": [self.staff_email]}):
```

Comment:
could you remind me what this does again?

Reply:
the fixture is like putting a decorator on every test in the testcase